src/components/IconButton/IconButton.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import "./IconButton.scss";
+
+type IconButtonProps = {
+  direction: "left" | "right";
+  onClick?: () => void;
+  label: string;
+  icon: React.FunctionComponent<React.SVGProps<SVGSVGElement>>;
+};
+
+function IconButton(props: IconButtonProps) {
```

Comment:
Is it maybe possible to reuse MenuButton for this one (or replace MenuButton with IconButton) so we don't have duplicate code?